--- setup.py~	2005-12-23 03:39:03.690947224 +0100
+++ setup.py	2005-12-23 03:40:21.653095176 +0100
@@ -17,12 +17,12 @@
 
 PLY is extremely easy to use and provides very extensive error checking. 
 """,
-            licence="""Lesser GPL (LGPL)""",
-            version = "1.3.1",
+            license="""Lesser GPL (LGPL)""",
+            version = "1.6",
             author = "David Beazley",
-            author_email = "beazley@cs.uchicago.edu",
+            author_email = "dave@dabeaz.com",
             maintainer = "David Beazley",
-            maintainer_email = "beazley@cs.uchicago.edu ",
-            url = "http://systems.cs.uchicago.edu/ply/",
+            maintainer_email = "dave@dabeaz.com",
+			url = "http://www.dabeaz.com/ply/",
             py_modules = ["lex","yacc"],
             )
